rekognition-apis/2-content-moderation.ipynb

```diff
@@ -0,0 +1,397 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Explicit, suggestive and violent content detection using Amazon Rekognition"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "This notebook provides a walkthrough of [content moderation APIs](https://docs.aws.amazon.com/rekognition/latest/dg/moderation.html) in Amazon Rekognition. You can quickly identify inappropriate content in your video and image libraries.\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Initialize stuff"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialise Notebook\n",
+    "import boto3\n",
+    "from IPython.display import Image as IImage, display\n",
+    "from IPython.display import HTML, display\n",
+    "from PIL import Image, ImageDraw, ImageFont\n",
+    "import time\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Curent AWS Region. Use this to choose corresponding S3 bucket with sample content\n",
+    "\n",
+    "mySession = boto3.session.Session()\n",
+    "awsRegion = mySession.region_name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Init clients\n",
+    "rekognition = boto3.client('rekognition')\n",
+    "s3 = boto3.client('s3')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# S3 bucket that contains sample images and videos\n",
+    "\n",
+    "# We are providing sample images and videos in this bucket so\n",
+    "# you do not have to manually download/upload test images and videos.\n",
+    "\n",
+    "bucketName = \"aws-rek-immersionday-\" + awsRegion"
```

Comment:
This is probably ok to start with in this PR, but maybe we should move to how we did it in the virtual proctoring folder, having a step 0 to create a supporting bucket to make the notebooks to run in all regions and remove dependencies from other buckets? I think the advantage is that if you wanna start playing around, by owning the bucket you can see its content, you can change it, you can learn better